README.md

```diff
@@ -38,39 +38,43 @@ sha1: 50bebedad9e0f10746a52b07239e47fa6c284d03 (Alternate)
 
 Congratulations, you are now sailing with the Ship of Harkinian! Have fun!
 
-## Additional Setup Information
+## About Versions
+
+The Ship of Harkinian's versioning system is a unique system, comprising of a three-numbered version (`x.y.z`) and a version codename.  The main version number `x` signifies major feature updates that require a new `oot.otr` file.  The version number `y` signifies major feature updates and bugfixes that most likely do not require a new `oot.otr` file, although it may be a good idea to create a new one.  The version number `z` signifies minor updates such as a collection of bugfixes.
```

Comment:
`y` versions will not require a new OTR period. Might be useful leaning more into the wording I've used for the #setup-guide channel from Discord.

Reply:
Yeah, it should be consistent across all documentation. Valid point.